examples/normal_sghmc.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""Correlated normal posterior. Inference with stochastic gradient Hamiltonian
+Monte Carlo.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import tensorflow as tf
+import numpy as np
+from matplotlib import pyplot as plt
+plt.style.use("ggplot")
+
+from edward.models import Empirical, MultivariateNormalFull
+
+# Helper functions
+# TODO: Where should these go?
+
+def mvn_plot_contours(z, label=False, ax=None):
+    """
+    Plot the contours of 2-d Normal or MultivariateNormalFull object.
+    Scale the axes to show 3 standard deviations.
+    """
+    # TODO: Make a plotting utils module?
+    sess = ed.get_session()
+    mu = sess.run(z.mu)
+    mu_x, mu_y = mu
+    Sigma = sess.run(z.sigma)
+    sigma_x, sigma_y = np.sqrt(Sigma[0,0]), np.sqrt(Sigma[1,1])
+    xmin, xmax = mu_x - 3 * sigma_x, mu_x + 3 * sigma_x
+    ymin, ymax = mu_y - 3 * sigma_y, mu_y + 3 * sigma_y
+    xs = np.linspace(xmin, xmax, num = 100)
+    ys = np.linspace(ymin, ymax, num = 100)
+    X, Y = np.meshgrid(xs, ys)
+    T = tf.convert_to_tensor(np.c_[X.flatten(), Y.flatten()], dtype=tf.float32)
+    Z = sess.run(tf.exp(z.log_prob(T))).reshape((len(xs), len(ys)))
+    if ax is None:
+        fig, ax = plt.subplots()
+    cs = ax.contour(X, Y, Z)
+    if label:
+        plt.clabel(cs, inline=1, fontsize=10)
+
+def retrieve_samples(qz):
```

Comment:
It took a bit of work to figure out how to grab the actual MCMC samples. Is there a method to do this that we missed, or should we move this elsewhere to allow for more general use?